WBooks/LIbrary/MainMenuController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  MainMenu.swift
+//  WBooks
+//
+//  Created by Gaston Maspero on 22/05/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class MainMenuController: UIViewController, UITableViewDataSource, UITableViewDelegate {
```

Comment:
When possible, set delegates and datasource in class extension, so you separate class's responsibilities